vpdq/cpp/io/vpdqio.cpp

```diff
@@ -176,6 +176,51 @@ bool readVideoDuration(
   durationInSec = (double)pFormatCtx->duration / MILLISEC_IN_SEC;
```

Comment:
nit: add note that `readVideoDuration` is not used atm (or remove it, but I think it could be useful eventually, say if we wanted to use it later to optionally print a set of debugging information)